src/main/java/seedu/address/model/task/Task.java

```diff
@@ -0,0 +1,135 @@
+package seedu.address.model.task;
+
+import static java.util.Objects.requireNonNull;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+import seedu.address.model.employee.Employee;
+
+/**
+ * Represents a task which belongs to a certain project.
+ */
+public class Task {
+
+    public static final String MESSAGE_CONSTRAINTS = "Tasks must be bound to a project first"
+        + " and later assigned to an Employee";
```

Comment:
There are cases where this message is being output even though it does not make sense such as empty inputs after prefixes. Perhaps either the message can be more generalised or each individual issue can have it's own message?

Reply:
<img width="851" alt="image" src="https://github.com/AY2324S1-CS2103T-T08-3/tp/assets/64006665/102e2bfa-9281-4f3e-85ea-892d2bb5ef28">


I've generalised the message - mainly that a name and deadline is compulsory and the format for adding a deadline.